### DISCUSSION TITLE
Got 6666666 RareTron.io in my account somehow....are these part of the course??

### DISCUSSION THREAD

**Discussion by Raghav-Programmer** [Cyfrin/foundry-full-course-cu#351] [Q&A] [answered]
I was trying to see the transactions of my metamask account on etherscan......There are three sections Transactions, Token Transfers, NFT Transfer......in token transfer I found 6666666 RareTron.io tokens somehow transferred to my account....I imported them to my account.......I created the metamask just for this course so I thought these might be related to this course.....is it true???

![tokesn](https://github.com/Cyfrin/foundry-full-course-f23/assets/113235744/c1b1bacb-577f-4218-b14b-f0db8ac4e348)

**Comment by ghost** [comment, marked as answer]
That's a scam airdrop, get used to seeing random NFTs or ERC20 appearing in your wallet. 

Probably the scam consist in making you visit that site to make you connect your wallet to drain your funds or steall other type of information, if you check the contract on [Sepolia Etherscan](https://sepolia.etherscan.io/address/0x54fa517f05e11ffa87f4b22ae87d91cec0c2d7e1#code) you will see only the bytecode so we don't know what the methods in the contract does.

Just don't interact with the tokens and hide them from Metamask.

**Reply by shikhar229169** [reply]
How these scammers manage to send it to every address.
I also got this 2 weeks ago.

**Reply by ghost** [reply]
I don't know how they do it.

**Reply by Raghav-Programmer** [reply]
ok thanks for answering.....I hid them from my metamask wallet.
